src/EnergyPlus/BranchNodeConnections.hh

```diff
@@ -50,7 +50,6 @@
 
 // ObjexxFCL Headers
 #include <ObjexxFCL/Array1D.hh>
-#include <ObjexxFCL/Optional.hh>
```

Comment:
The second most common change in this PR is the removal of includes that aren't used.